src/components/GenuineCheck.js

```diff
@@ -157,9 +160,9 @@ class GenuineCheck extends PureComponent<Props> {
         id: 'deviceInfo',
         title: (
           <Trans i18nKey="deviceConnect.dashboard" parent="div">
-            {'Navigate to the '}
-            <Bold>{'dashboard'}</Bold>
-            {' on your device'}
```

Comment:
1. Keep as is
2. dashboard -> Dashboard

Reply:
done